cmd/copy.go

```diff
@@ -638,6 +640,11 @@ func (raw rawCopyCmdArgs) cook() (CookedCopyCmdArgs, error) {
 		cooked.preserveSMBInfo = false
 	}
 
+	cooked.preservePOSIXProperties = raw.preservePOSIXProperties
+	if cooked.preservePOSIXProperties && !areBothLocationsPOSIXAware(cooked.FromTo) {
```

Comment:
Nit: interesting that the if condition in go doesnt seem to be enclosed in parenthesis - is this the only syntax in golang for conditions or do they also support paranthesis?

Reply:
They do support parenthesis-- But it's idomatic to not use it around the main conditional.